### PR TITLE
Madhav Pull request

### PR DESCRIPTION
**Pull request by BCT-Madhav**
I hope I had completed as stated in case of any error contact through mail at 077bct041.madhav@pcampus.edu.np

### CODE REVIEW

**Review by ABoredComputerEngineer — CHANGES_REQUESTED** [review]
Looks like you made a typo in  your email address. Could you please look into that?

**Comment by BCT-Madhav** [comment]
Bujina ni daju maile

**Comment by BCT-Madhav** [comment]
I have corrected the error in the email address. Thank you for the response. If any further error are left please inform me.
Thank You 
Madhav Prasad Parajuli
[BCT-Madhav.txt](https://github.com/IT-Club-Pulchowk/Git-Workshop/files/6500600/BCT-Madhav.txt)

File: Stories/BCT-Madhav.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name: Madhav Prasad Parajuli
+email: 077bct41.madhav@pcampus.edu.np
```
Comment (ABoredComputerEngineer) [inline Stories/BCT-Madhav.txt:2]: This should be `077bct041.madhva@pcampus.edu.np`.

**Review by ABoredComputerEngineer — CHANGES_REQUESTED** [review]
Please check your email again. Looks like you missed a '0'.

**Comment by BCT-Madhav** [comment]
Sorry for the last error it is solved now 
[BCT-Madhav.txt](https://github.com/IT-Club-Pulchowk/Git-Workshop/files/6500818/BCT-Madhav.txt)

**Comment by ABoredComputerEngineer** [comment]
Thank you for your submission! Have a nice day!

**Pull request merged**